packages/flutter/lib/src/material/elevation_overlay.dart

```diff
@@ -62,10 +66,26 @@ class ElevationOverlay {
   ///    specifies the exact overlay values for a given elevation.
   static Color overlayColor(BuildContext context, double elevation) {
     final ThemeData theme = Theme.of(context);
+    return _overlayColor(theme.colorScheme.onSurface, elevation);
+  }
+
+  /// Returns a color blended by laying a semi-transparent overlay (using the
+  /// [overlay] color) on top of a surface (using the [surface] color).
+  ///
+  /// The opacity of the overlay depends on [elevation]. As [elevation]
+  /// increases, the opacity will also increase.
+  ///
+  /// See https://material.io/design/color/dark-theme.html#properties.
+  static Color colorWithOverlay({required Color surface, required Color overlay, required double elevation}) {
```

Comment:
Usually named parameters get a leading space, like `({ Color foo, Color bar })` or line breaks:

```dart
static Color colorWithOverlay({
  required Color surface, 
  required Color overlay, 
  required double elevation,
}) {
```

It seems a little odd to have three required named parameters. The similar elevationOverlay static method has 3 positional parameters. Do we expect this method to gain default parameter values or more parameters?
```

Reply:
Good point, I'll make these positional parameters, I do not expect default values or additional parameters to be added to this method.